pep/app/pods/components/page/sidebar/widgets/whats-new/component.ts

```diff
@@ -49,7 +50,9 @@ export default class PageSidebarWidgetsWhatsNew extends Component<PageSidebarWid
     @action
     @dontRunInFastboot
     onElementInsert() {
-        taskFor(this.loadResults).perform();
+        setTimeout(() => {
+            taskFor(this.loadResults).perform();
+        }, 1000);
```

Comment:
i know this is probably only temporary, but we should use `later(() => ..., 1000);`